306192 kn0427 (#39)
osisoft/PI-Adapter-Modbus-Docs: content/configuration/security.md

---
uid: PIAdapterForModbusSecurity
---

# Security 

> When determining Modbus TCP security practices with regards to REST APIs, you should consider the following practice. To keep the adapter secure, it should only be installed on machines that restrict access to administrators only. REST APIs are bound to localhost, meaning that only requests coming from within the machine will be accepted. 

Reviewer: ```suggestion
When determining Modbus TCP security practices with regards to REST APIs, you should consider the following practice. To keep the adapter secure, only administrators should have access to machines where the adapter is installed. REST APIs are bound to localhost, meaning that only requests coming from within the machine will be accepted. 
```
